### PR TITLE
API-9074: scopes for Reports API

### PR DESCRIPTION
**Pull request by grz-gajda**
# 🚀 Links

Replace the placeholder data in links 👇

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-9074)
- [Jira](https://livechatinc.atlassian.net/browse/API-9074)

# 📓 Description

Provides information about required scopes for Reports API.

### CODE REVIEW

**Review by janowsiany — APPROVED** [review]

**Comment by oliwiapolec** [comment]
Hey, as discussed on Slack, please add a note about this change in the changelog :) Since we're changing request validation, we should inform our customers about it, even if we use already existing resources (in this case the reports_read scope).

We can add it in the [general](https://developers.livechat.com/docs/data-reporting/changelog/#general) section. The note can be similar to this one: https://developers.livechat.com/docs/management/changelog/#agents
and by that I mean we can mention the change applies to versions 3.x :)

File: content/data-reporting/changelog/index.mdx [hunk 1]
```diff
@@ -50,6 +50,7 @@ The developer preview version provides a preview of the upcoming changes to the
     - `filters.surveys.type`
     - `filters.surveys.answer_id`
     - `filters.events.types`
+- All methods requires the `reports_read` scope. This applies to **v3.** versions of the _Reports API_.
```
Comment (oliwiapolec) [inline content/data-reporting/changelog/index.mdx:53]:
```suggestion
- All methods require the `reports_read` scope. This applies to all **v3.x** versions of the _Reports API_.
```

Comment (oliwiapolec) [reply content/data-reporting/changelog/index.mdx:53]: I'd also move this note above the renamed fields list, under this point:
> To provide more consistent experience across all Platform APIs, filters are now aligned with those used in List Archives (the Agent Chat API).

it gets "lost" when it's under a long list like that :D

Comment (grz-gajda) [reply content/data-reporting/changelog/index.mdx]: Corrected.

**Review by oliwiapolec — APPROVED** [review]

**Pull request merged**